config/routes.rb

```diff
@@ -5,18 +5,11 @@
   get '/auth/:provider/callback', to: 'sessions#create'
   get '/logout', to: 'sessions#destroy'
 
-  get '/registration', to: 'users#new'
-  post '/registration', to: 'users#create'
-
-  get '/dashboard/:user_id', to: 'users#dashboard', as: '/dashboard'
-
```

Comment:
Including the `user_id` was redundant because we have the `current_user` method that can populate the current user's info.  Updated by creating a Dashboard Controller with an index action.

We'll need to update the redirect page in the Google OAuth setup

Reply:
We should fire up two local servers and test this out.  Unless I'm missing something, I think it should still route correctly (the callback to Oauth is defined in the welcome route to `Sign In With Google`, and then we just redirect to `dashboard_path ...` after the callback to get the token is completed.  In other words, I think the registered callback URI remains the same, and these are just redirect actions we're controlling after we've gotten what we need from Oauth

Reply:
> We should fire up two local servers and test this out.

@tvaroglu This was working for me locally when I was running localhost on the FE and using the heroku BE.  I had just noticed the `#` at the end of the path, so wasn't sure whether that was a localhost thing or a google redirect thing.